src/core/lib/experiments/config.cc

```diff
@@ -31,7 +31,11 @@
 #include <grpc/support/log.h>
 
 #include "src/core/lib/config/config_vars.h"
+#ifndef GRPC_EXPERIMENTS_TEST_FIXTURE
 #include "src/core/lib/experiments/experiments.h"
+#else
+#include "test/core/experiments/test_experiments.h"
```

Comment:
I don't think this is the right solution, `#include`ing a singular, fixed test file in production code. What we need here is a way to inject configuration into the configuration parser ... some way to separate the loading of config data, from config processing & accessing.

Reply:
Took a different approach to injecting test experiments now with a slightly modified approach.